rubin_sim/maf/metrics/uniformity_metrics.py

```diff
@@ -0,0 +1,91 @@
+__all__ = "LinearMultibandModelMetric"
+
+import numpy as np
+from rubin_sim.maf.metrics.base_metric import BaseMetric
+from rubin_sim.maf.metrics.exgal_m5 import ExgalM5
+import healpy as hp
+
+
+class LinearMultibandModelMetric(BaseMetric):
+    """ """
+
+    def __init__(
+        self,
+        model_dict,
+        metric_name="LinearMultibandModel",
+        m5_col="fiveSigmaDepth",
+        filter_col="filter",
+        post_processing_fn=lambda x : x,
+        units="mag",
+        extinction_cut=0.2,
+        min_depth_cut={'i': 25.0},
+        max_depth_cut={},
+        mean_depth={"u": 0.0, "g": 0.0, "r": 0.0, "i": 0.0, "z": 0.0, "y": 0.0},
+        n_filters=6,
+        **kwargs,
+    ):
+        """
+        Args:
```

Comment:
Could you please add docstrings in the LSST developer style (numpydoc style)? https://developer.lsst.io/python/numpydoc.html

Reply:
I've just added docstrings for most (if not all) functions/classes